skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -107,6 +107,7 @@ Example 5 returns information for user accounts that have been assigned a design
 Indicates the Identity of the user account to be retrieved.
 
 For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account:
+(Note: The mentioned changes are currently applicable only for Tenants in Public Cloud without a 
```

Comment:
@isrumnon - seems like an incomplete sentence, also instead of Public Cloud - use Commercial Environments only like done below.

Reply:
done